python/hsfs/feature_group.py

```diff
@@ -468,7 +468,7 @@ def __init__(
         self._feature_store_name = featurestore_name
         self._description = description
         self._created = created
-        self._creator = creator
+        self._creator = user.User.from_response_json(creator)
```

Comment:
Did you test this? the `creator` field is available only if the data comes from the backend. If the class is initialized by the user through the API `creator` is going to be null.

If you see the code below the one you changed, you see that we differentiate between the case in which the class is instantiated with data coming from the backend or data added by the user.

Reply:
if creator is null, then result of user.User.from_response_json(creator) is also None

Reply:
here is an example:

Reply:
![image](https://user-images.githubusercontent.com/24224279/128878715-ad1f5fe4-f23e-4151-be69-694430abb4ff.png)

Reply:
i could change it as follows, if it is still needed:
if creator is not None:
    self._creator = user.User.from_response_json(creator)
else:
    self._creator = None